### PR TITLE
<Popover/> - improve arrow styles

### PR DESCRIPTION
**Pull request by yardnsm**
This PR improves the `<Popover/>`'s arrow styles to match the `<Tooltip/>` design. It increases its size a bit and adds a shadow to it.

Old design:
screen shot 2018-12-16 at 15 03 09

New design:
screen shot 2018-12-16 at 15 03 20

### CODE REVIEW

File: src/Popover/Popover.st.css [hunk 1]
```diff
@@ -13,6 +18,10 @@
   -st-named: shadow30;
 }
 
+:vars {
+  contentArrowSize: 10px;
```
Comment (shlomitc) [inline src/Popover/Popover.st.css]: change to 8px (so the cross will be ~12px)

**Comment by alonbt** [comment]
@yardnsm @shlomitc @hepiyellow Can you please resolve this ? I need this fix in order to use the tooltip and deploy a feature...

**Comment by yardnsm** [comment]
@LittleTal The arrow spacing has been fixed.

**Regarding the minimum margin:** this can be easily implemented using a [Popper.js modifier](https://popper.js.org/popper-documentation.html#modifiers..preventOverflow.padding), but I think its better if we won't that to the core's `<Popover/>` directly.

Instead, we can add a `popperModfiers` prop to the core's `<Popover/>`, so we can control the modifiers  inside the WSR `<Popover/>`.

I think we can merge this PR without the minimum margin thing.

@shlomitc wdyt?

**Comment by shlomitc** [comment]
@yardnsm - if we can't achieve this with pure css, then let's merge this one just so it won't block @alonbt. then, let's add the margin capability to the core

**Pull request merged**